buildscripts/test.py

```diff
@@ -196,6 +199,56 @@
 
         format_print(f'Tests for {package_type} packages are PASSED')
 
+    # Execute examples
+    if test_mode == 'examples':
+        format_print(f'Run SDC examples')
+        sdc_examples = os.path.join(sdc_src, 'examples')
```

Comment:
I would advise to use `pathlib` module instead of `os` as a newer approach.

Reply:
I would like to leave os.path.join here due to changing this to pathlib require to make changes in all other places.
I think this should be done in separate PR.

Reply:
I'm not sure it will be done in separate PR. It was like a recommendation, so you we can skip it in this PR.